source/application/CLS/CLSAppController.h

```diff
@@ -12,8 +12,8 @@ class CLSAppController : public AMAppController
 public:
 	enum CLSBeamlineID
 	{
-		OtherFacility = 1,    //reserved for AMAppControlelr
-		PresevedBeamline = 2, //reserved for AMAppControlelr
+		BlankFacility = 1,            //reserved for AMAppControlelr
+		PreserverdALS801Beamline = 2, //reserved for AMAppControlelr
```

Comment:
I don't know. This exists in the legacy codes.
maybe @dretrex  can answer this question